.github/dependabot.yml

```diff
@@ -9,3 +9,7 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+    commit-message:
+      # Prefix all commit messages with "bump: "
+      prefix: "bump"
+      prefix-development: "bump"
```

Comment:
This `prefix-development` feature is supported only by bundler, composer, mix, maven, npm, and pip. I don't think, it may make a sense in cargo context.